tests/platform_tests/test_advanced_reboot.py

```diff
@@ -62,15 +85,29 @@ def test_fast_reboot_from_other_vendor(duthosts,  rand_one_dut_hostname, request
     flush_dbs(duthost)
     advancedReboot.runRebootTestcase()
 
+
 @pytest.mark.device_type('vs')
-def test_warm_reboot(request, get_advanced_reboot, verify_dut_health,
-    advanceboot_loganalyzer, capture_interface_counters):
+def test_warm_reboot(request, testing_config, get_advanced_reboot, verify_dut_health, duthosts, advanceboot_loganalyzer, 
+    capture_interface_counters, toggle_all_simulator_ports, rand_selected_dut, toggle_simulator_port_to_upper_tor):
```

Comment:
Be careful with what rand host fixture you use here and let's make sure that it si consistent with picking the same tor as `enum_rand_one_per_hwsku_frontend_hostname` in advanced_reboot. If these hosts are different then test outcome will be uncertain. 


https://github.com/sonic-net/sonic-mgmt/blob/fd1652e893c022744b27d25963a1d6ab9bc0ad87/tests/common/fixtures/advanced_reboot.py#L814

Reply:
Updated to use same fixture to keep it consistent. However in this case the host chosen does not matter as we are just using the duthost to get how many mux interfaces there are.